src/components/Layout/Header/GlobalSearch/AssetResults/AssetResult.tsx

```diff
@@ -51,12 +52,19 @@ export const AssetResult = forwardRef<AssetResultProps, 'div'>(
             >
               {asset.name}
             </RawText>
-            <RawText size='xs' variant='sub-text'>
-              {asset.symbol}
-            </RawText>
+            <Flex alignItems='center' gap={2}>
+              <RawText size='xs' variant='sub-text'>
+                {asset.symbol}
+              </RawText>
+              {asset.id && (
+                <RawText size='xs' variant='sub-text'>
+                  {middleEllipsis(asset.id)}
+                </RawText>
+              )}
+            </Flex>
           </Flex>
         </Flex>
-        {bnOrZero(fiatBalance).gt(0) && (
+        {bnOrZero(cryptoHumanBalance).gt(0) && (
```

Comment:
no market data so update to crypto balance to show nft balance values

Reply:
@reallybeard FYI ^^